### DISCUSSION TITLE
Time stamp for the request log :

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4091] [Ideas]
Hello, 

Would it be possible to add a feature to see the query log at a given time ?

Example : 

I want to see the query log for Jan 3, 2022 at 3:38 PM. 
So, I enter the date and time and access it directly. 

Thank you,

**Comment by ainar-g** [comment]
I think that there was a feature request about it.  We'll only be able to do that after some refactorings as well as once #2290 is resolved.
